src/index.js

```diff
@@ -0,0 +1 @@
+import './style.css';
```

Comment:
- [ ] At this point of the project you should have developed a working version of the application as the project requirements said. Kindly take a look at this screenshot and to project requirements. 😸 
    - All the buttons should be functional, which means, the submit button should add a score to the list, and the refresh button should refresh the list of scores 🚀
    - Remember to use ES6 syntax
    ![imagen](https://user-images.githubusercontent.com/60789317/136084215-0b0d88af-7940-43a2-84e5-4ab5548e93cf.png)

Reply:
Please kindly find attached a copy of the project requirements. We are to implement the changes you requested in the next project milestone. 
![requirements](https://user-images.githubusercontent.com/33420420/136184672-31d5f758-319a-4348-92d7-d4bc8454456e.png)

Reply:
I see you will implement the following tasks in the next milestone.